### DISCUSSION TITLE
🚀 Improve handling of email permissions to recieve email

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#4419] [Feature Requests]
### Suggest an idea for ExpressionEngine

There's a per member setting where they can opt out of receiving notification emails.
<img width="493" alt="Screenshot 2024-08-08 at 6 10 33 PM" src="https://github.com/user-attachments/assets/1dda32ac-95f6-4acc-9bbe-eac994a63783">

Problem/possible bug.  If you send from communicate to a member role, it doesn't take the setting into account.  I'm honestly not sure where it does/does not take the setting into account.  If they're on the 'notify' setting for channel entries?  If someone sends them an email directly to the address from 'Communicate'?  If they send it to a group? If we send them a security email when their password is changed?

Some- at least the security email- seems like you shouldn't be able to opt out of.

It wouldn't hurt to figure out WHERE this is being used and where it's not, decide where we WANT it to be used, and clearly document it.  That could involve extra settings- like a confirmation modal- in certain cases.

IDK.  But I think we need to be clear where those settings DON'T take affect, and perhaps have that reflected in both the docs and the verbiage on the setting.

**Comment by intoeetive** [comment]
I *think* (as much as I have investigated the code) that these settings actually are not respected when an email is sent from the CP. Some of the settings are used by Comment module and I think some others might be used by Forum when installed - also I don't see them in the code, but it might be more tricky to search

I think we need to clean this up, but maybe as part of revising settings in general. I think we still have few other pieces that are no longer relevant in EE7
